src/app/phase2/phase2.component.ts

```diff
@@ -12,9 +13,14 @@ import {DataService} from '../core/services/data.service';
 export class Phase2Component implements OnInit {
   public teamFilter = 'All Teams';
 
-  constructor(public userService: UserService, private dataService: DataService) {}
+  constructor(public userService: UserService, private dataService: DataService, private issueService: IssueService) {}
 
-  ngOnInit() {}
+  ngOnInit() {
+    // For Students, they do not have team filter, so the team filter is initialised here.
+    if (this.userService.currentUser.team !== undefined) {
+      this.issueService.setIssueTeamFilter(this.userService.currentUser.team.id);
```

Comment:
Since the default filter is the teams/team in which the user is in, is it possible to assign the default filter in the service, when the team/teams are determined?

Reply:
I have moved this code to `getTeamFilterString()`, I couldn't move it to `UserService` as there will be circular dependency